core/misc/ckeditor/lang/af.js

```diff
@@ -1,5 +1,5 @@
-﻿/*
-Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
-For licensing, see LICENSE.md or http://ckeditor.com/license
-*/
+﻿/*

```

Comment:
are all of the `core/misc/ckeditor/lang/` file changes the same? and what has actually changed, I can't tell...must be some esoteric Git thing?

Reply:
maybe end-of-line markers?